NewSpring/corinth
# chore: Prayer Card Changes

## DESCRIPTION

### What does this PR do, or why is it needed?
This is setting up the prayer card so that it shows just the necessary information when a person gets to a prayer from a notification that the prayer was answered. 

Added a `readOnly` navigation prop that will tell us whether or not the prayer card should or should not have all the extra bits. If `readOnly` is true, then it should remove:

* The "Save Prayer" button
* The "I've Prayed" and "Report Prayer" buttons
* The "Praying For" header in the top left of the screen
* The "How to pray" link

Before:
![Screenshot 2020-07-22 11 37 43](https://user-images.githubusercontent.com/3229463/88198567-d5b2da00-cc11-11ea-8751-1a3d922e37e6.png)

After:
![Screenshot 2020-07-22 11 37 24](https://user-images.githubusercontent.com/3229463/88198595-dea3ab80-cc11-11ea-906d-5fb5e5777d52.png)

### How do I test this PR?
The default for `readOnly` is `false`, so if you run this branch and go look at a prayer, everything should look the same as it does now.

To test whether or not the appropriate stuff gets removed, you will have to change the default for the `readOnly` prop (Line 76) to be `true`. Then go look at a prayer.

Eventually, the prop will come from a link but that is beyond the scope of this PR.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
